Gemfile

```diff
@@ -6,8 +6,8 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 ruby '~> 2.7.0'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails', branch: 'main'
-gem 'rails', '~> 6.1.3', '>= 6.1.3.1'
 gem 'jquery-rails'
+gem 'rails', '~> 6.1.3', '>= 6.1.3.1'
```

Comment:
It doesn't look like this needs to be here. And if it does, you should remove the comment on line 8 or move it.